credentials/alts/alts_test.go

```diff
@@ -287,6 +296,50 @@ func (s) TestCheckRPCVersions(t *testing.T) {
 	}
 }
 
+func TestFullHandshake(t *testing.T) {
```

Comment:
A comment explaining what the test is doing, and what it is verifying would be very helpful to new readers of the code.

Reply:
Done.